app/Http/Controllers/ProfileController.php

```diff
@@ -82,6 +85,20 @@ private function saveAvatar($request){
     }
 
     public function destroyProfile(Request $request){
+        Auth::user()->meetings()->forceDelete();
+        Auth::user()->joinMeetings()->detach();
+        if(Auth::user()->avatar){
+            $this->deleteAvatar(Auth::user()->avatar);
+        }  
+        Auth::user()->participant()->forceDelete();
+        $reviews = Contact::where('subtitle_id', 5)->where('email', Auth::user()->email)->get();
+        if($reviews){
```

Comment:
no need for if statement here since the `$reviews` will always have a value based on your query